web/src/components/Diagram/components/Timeline/Timeline.tsx

```diff
@@ -1,13 +1,18 @@
 import {ParentSize} from '@visx/responsive';
 
 import {IDiagramComponentProps} from 'components/Diagram/Diagram';
+import {noop} from 'lodash';
+import Controls from '../DAG/Controls';
 
 import Visualization from './Visualization';
 
 const Timeline = (props: IDiagramComponentProps) => (
-  <ParentSize parentSizeStyles={{height: 'auto', overflowY: 'scroll'}}>
-    {({width}) => <Visualization {...props} width={width} />}
-  </ParentSize>
+  <>
+    <Controls isMiniMapActive={false} onMiniMapToggle={noop} isTimeline />
```

Comment:
What do you think about having default for the onMiniMapToggle and isMiniMapActive props as well

Reply:
ok